Terminal.Gui/View/Layout/ViewLayout.cs

```diff
@@ -560,7 +560,7 @@ internal void OnResizeNeeded ()
 		var relativeBounds = SuperView is { IsInitialized: true } ? SuperView.Bounds :
 			Application.Top != null && Application.Top.IsInitialized ? Application.Top.Bounds :
 										   Application.Driver?.Bounds ??
-										   new Rect (0, 0, int.MaxValue, int.MaxValue);
+										   Rect.Empty;
```

Comment:
Please leave this as it was. This will be important for `Dim.Auto`.

Reply:
Done.